purchases/src/androidTestIntegrationTest/kotlin/com/revenuecat/purchases/BasePurchasesIntegrationTest.kt

```diff
@@ -49,10 +49,10 @@ open class BasePurchasesIntegrationTest {
     protected val testUserId = "android-integration-test-$currentTimestamp"
     protected val proxyUrl = Constants.proxyUrl.takeIf { it != "NO_PROXY_URL" }
 
-    protected lateinit var mockBillingAbstract: BillingAbstract
+    internal lateinit var mockBillingAbstract: BillingAbstract
 
-    protected var latestPurchasesUpdatedListener: BillingAbstract.PurchasesUpdatedListener? = null
-    protected var latestStateListener: BillingAbstract.StateListener? = null
+    internal var latestPurchasesUpdatedListener: BillingAbstract.PurchasesUpdatedListener? = null
+    private var latestStateListener: BillingAbstract.StateListener? = null
```

Comment:
Had to change the visibility of these after #1106, otherwise it wouldn't compile